custom/mongo-utils/buildscripts/push

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+if [ ! -z "${DNAME}" ] && [ ! -z "${DPASS}" ];
+then
+
+  docker login -u "${DNAME}" -p "${DPASS}";
+  #Push to docker hub repository with latest tag
+  docker buildx build . -f Dockerfile --progress plane --push --no-cache --platform linux/amd64,linux/arm64 --tag litmuschaos/mongo-utils:4.0.5
```

Comment:
litmuschaos/mongo-utils:4.0.5 Is there any specific tag only needed ? Or we can have latest tag as other images. PTAL in Makefile, used latest one.

Reply:
No, this is the specific tag.